### DISCUSSION TITLE
How to get chatgpt scores on ppl tasks

### DISCUSSION THREAD

**Discussion by RanchiZhao** [open-compass/opencompass#462] [Q&A] [answered]
I think we can not get the logprobs from models like gpt-3.5-turbo/GPT4, so how do you eval those ppl tasks.

**Comment by Leymore** [comment, marked as answer]
In fact, we indeed cannot evaluate the PPL task on API models like ChatGPT. For API models, we always use the Gen evaluation method.

**Reply by RanchiZhao** [reply]
thanks a lot!
